.github/actions/people/people.py

```diff
@@ -0,0 +1,612 @@
+"""Use the github API to get lists of people who have contributed in various ways to Pydantic.
+
+This logic is inspired by that of @tiangolo's
+[FastAPI people script](https://github.com/tiangolo/fastapi/blob/master/.github/actions/people/app/main.py).
+"""
+import requests
+import logging
+import subprocess
+import sys
+from collections import Counter
+from datetime import datetime, timedelta, timezone
+from pathlib import Path
+from typing import Any, Container, Dict, List, Set, Union
+
+import yaml
+from github import Github
+from pydantic import BaseModel, SecretStr
+from pydantic_settings import BaseSettings
+
+github_graphql_url = "https://api.github.com/graphql"
+
+discussions_query = """
```

Comment:
Hello

I created a package for generate a GraphQL query from python code based on pydantic.
Home page: https://github.com/denisart/graphql-query
docs: https://denisart.github.io/graphql-query/usage.html

What do you think about creating queries in graphql-query format? And will it be possible to use packages that depend on pydantic?